ocs_ci/helpers/tlsprofile_helper.py

```
@@ -1145,3 +1536,573 @@ def _cleared():
+    Wait until the workload is ready. When ``previous_fingerprints`` is set,
+    also wait until none of those fingerprints remain (new pod UID, restart,
+    or container start time).
+    """
+    log.info(
+        "Waiting for %s to be ready after TLSProfile change "
+        "(previous fingerprints=%d)",
+        workload_name,
+        len(previous_fingerprints or ()),
+    )
+
+    def _done():
```

Ruff (0.16.2)

[warning] Missing return type annotation for private function `_done`

(ANN202)